src/codecs/webp/vp8.rs

```diff
@@ -1903,11 +1903,8 @@ fn predict_4x4(ws: &mut [u8], stride: usize, modes: &[IntraMode], resdata: &[i32
                 IntraMode::HU => predict_bhupred(ws, x0, y0, stride),
             }
 
-            // Create a [i32; 16] array for add_residue by copying the
-            // slice from resdata into rb (slices do not work).
-            let mut rb = [0i32; 16];
-            rb.copy_from_slice(&resdata[i * 16..i * 16 + 16]);
-            add_residue(ws, &rb, y0, x0, stride);
+            let rb: &[i32; 16] = resdata[i * 16..][..16].try_into().unwrap();
+            add_residue(ws, rb, y0, x0, stride);
```

Comment:
Do we know whether this is faster? The entire array fits in a single cache line, so copying it is extremely fast and this may not be a net win

Reply:
Comparing the previous and newer version when using `opt-level=3` generates the same number of instructions
1.54: https://rust.godbolt.org/z/hGf7GYn4r
1.34: https://rust.godbolt.org/z/Y669fv3h4

While compiling with a lower `opt-level` generates less instructions for the `.try_into()` version
1.54: https://rust.godbolt.org/z/fda9r8rKa
1.34: https://rust.godbolt.org/z/j6joTovj5

so to me this seems worth it.